chat.delta.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Name=Delta Chat
+Comment=Delta.chat
+Exec=/app/bin/run.sh
```

Comment:
you don't have to enforce the bin path here. Just use a "correct" filename like `delta-desktop`